src/psyclone/domain/lfric/__init__.py

```diff
@@ -62,6 +61,8 @@
     MetadataToArgumentsRules
 from psyclone.domain.lfric.arg_index_to_metadata_index import \
     ArgIndexToMetadataIndex
+from psyclone.domain.lfric.lfric_kern import LFRicKern
+from psyclone.domain.lfric.lfric_kern_call_factory import LFRicKernCallFactory
 from psyclone.domain.lfric.lfric_collection import LFRicCollection
```

Comment:
Can do - I understood that these needed to be imported in a specific order to avoid circular dependencies, but will try alphabetising and see if anything falls over

Reply:
As suspected - pytest falls over with an ImportError before getting to the tests when these get swapped

Reply:
Lines 41-44 already say this:
```
# The order here is not alphabetical, but important because
# there are various dependencies between the modules (e.g.
# KernCallAccArgList imports KernCallArgList, ArgOrdering
# imports LFRicArgDescriptor, ...).```